CMakeSettings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-release",
+      "generator": "Ninja",
+      "configurationType": "Release",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "C:\\github repros\\out\\freecadbuild\\${name}",
+      "installRoot": "C:\\github repros\\out\\freecadinstall\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "FREECAD_LIBPACK_DIR",
+          "value": "C:/freecad_libs/LibPack",
+          "type": "PATH"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
This entire file should not be a part of this PR. Please remove this from the commit.

Reply:
Totally right I missed that! Is deleted.

Reply:
Thank you. But as I mentioned earlier, remove this from the history itself, not just by adding a commit that does the deletion. You can do that by squashing the deletion commit with the one that added the file.

Reply:
I am on windows on github desktop. I see no buttons, please don't make me go to console ;)